src/utils/uiHelpers.ts

```diff
@@ -0,0 +1,64 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+import * as vscode from 'vscode';
+
+export interface APIResource {
+  readonly name: string;
+
+  readonly title: string;
+  readonly placeholder: string;
+  readonly ignoreFocusOut?: boolean;
+
+  fetchItems(query?: string): Promise<vscode.QuickPickItem[]>;
+}
+
+export class APIQuickPick {
```

Comment:
Would have expected this to be inside the terraformCloud folder as while it's a generic VSCode UI thing, its only really going to be used by the  tfc stuff